controllers/postController.js

```diff
@@ -66,31 +68,36 @@ const list = async (req, res) => {
   }
 
   const keywordId = req.params.keywordId;
-  const { includedKeyword, cursorId } = req.query;
+  const { includedKeyword, excludedKeyword, cursorId } = req.query;
+  const includedKeywordList = includedKeyword.split(",").join("|");
+  const excludedKeywordList = excludedKeyword.split(",").join("|");
   const limit = Number(req.query.limit);
   let hasNext = false;
-  let postListResult;
+  const contentFilter =
+    excludedKeywordList.length === 0
+      ? {
+          $regex: includedKeywordList,
+        }
+      : {
+          $regex: includedKeywordList,
+          $not: { $regex: excludedKeywordList },
+        };
 
   try {
-    if (isEmptyString(cursorId)) {
-      postListResult = await postModel
-        .find({ keywordId })
-        .find({ content: { $regex: includedKeyword } })
-        .sort({ _id: -1 })
-        .limit(limit);
-    } else {
-      postListResult = await postModel
-        .find({ keywordId })
-        .find({ content: { $regex: includedKeyword } })
-        .find({ _id: { $lt: cursorId } })
-        .sort({ _id: -1 })
-        .limit(limit);
-    }
+    const postListResult = await postModel
+      .find({ keywordId })
+      .find({
+        content: contentFilter,
+      })
+      .sort({ _id: isEmptyString(cursorId) ? -1 : { $lt: cursorId } })
```

Comment:
[공식문서](https://www.mongodb.com/ko-kr/docs/manual/reference/operator/aggregation/sort/#std-label-sort-pipeline-metadata)에서는 `sort()` 내부에 `$lt`와 같은 조건을 사용할 수 있다고 명시되어 있지 않는데, `cursorId`가 있을 때 해당 조건이 정상적으로 작동하나요?!

Reply:
```suggestion
  try {
    if (isEmptyString(cursorId)) {
      postListResult = await postModel
        .find({ keywordId })
        .find({ content: contentFilter })
        .sort({ _id: -1 })
        .limit(limit);
    } else {
      postListResult = await postModel
        .find({ keywordId })
        .find({ content: contentFilter })
        .find({ _id: { $lt: cursorId } })
        .sort({ _id: -1 })
        .limit(limit);
    }
```
감사해요 한솔님. 제가 재사용 로직을 통합하면서 놓쳤네요! 
`isEmptyString(cursorId)` 케이스만 통과되어서 문제가 없었는데, 막 클라이언트 개발하면서 보니 오류가 발생했어요. 위 처럼 다시 분기 처리하는 것으로 수정했습니다.